### PR TITLE
Desktop view Update

### PR DESCRIPTION
**Pull request by yawsamcode**
I added various parts of the desktop version with a responsive layout.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -126,38 +131,59 @@ <h2>Profesional Art Printing Data</h2>
...
+            <!--<div class="see-project-btn" id="long-btn">See Project</div>-->
+        </section>
+    </div>
+            
+</section>
+</section>
+
         <section class="about-me-section">
             <div class="aboutme-head">
                 <h2 class="about-me-title">About Me</h2>
```
Comment (Shedrack-Sunday) [inline index.html:189]: - [ ] Good job working on this feature 👏 . However, the placement of the background image in your design needs a little adjustment to tally with that of the one given on the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=I0SUZ5FGkB27z6XP-0) Template. The reason for emphasizing this Is to help you build a programming muscle in meeting the requirements as given, which is the medium clients or your manager will follow. I've made a comparison below for your review 

## Figma
image

## Your design:
image

File: index.html [hunk 2]
```diff
@@ -126,38 +131,59 @@ <h2>Profesional Art Printing Data</h2>
...
+        <section class="project-sample" class="card">
+            <div class="project-details">
+                <h2>Profesional Art Printing Data</h2>
+                <article>A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    has been the industry's standard</article>
+            </div>
+            <div class="stack-used">
+                <p class="stack-style2" id="stack-margin">html</p>
+                <p class="stack-style2" id="stack-margin">bootstrap</p>
+                <p class="stack-style2" id="stack-margin">Ruby</p>
```
Comment (Shedrack-Sunday) [inline index.html:178]: - [ ] Nice efforts with the card 👏 . However, I noticed two issues that you need to fix.
- First, the background image of the cards on the hover effect is meant to be white, and the button is at the button, not the top. kindly fix this 👍 
- Secondly, the first card I a reference to all other cards. I strongly suggest you implement the same features in the first point above on all cards 👍 

- [ ] The reason for asking for these changes in reference to the template, is to master the art of sticking to user requirements, and have a good we app that gives a better user experience 👍 

## Your view:
image

## Figma guide:
image

File: index.html [hunk 3]
```diff
@@ -26,7 +27,20 @@ <h1 class="name">Sam</h1>
...
+                <li><a href="#"></a><img src="./images/angel list.svg" alt="Angel_List" class="angel-list"></li>
+                <li><a href="https://twitter.com/sammi_108"></a><img src="./images/twitter.svg" alt="Twitter" class="twitter"></a>
+                </li>
+                <li><a href="#"><img src="./images/medium.svg" alt="Medium" class="medium"></a></li>
+            </ul>
+            </section>
+            <section class="heading-to-right">
             <h1>Hey There. <br>
                 <h1>I’m Samuel <br></h1>
                 <h2>I’m a software developer!</h2>
```
Comment (Shedrack-Sunday) [inline index.html:46]: - [ ] For this section of the desktop version, kindly adjust the background image to cover up the section it occupies just as provided on the template. this gives a more visual appealing view, and heeps you keep to the rquiremnts provided 👍

## Your view:
image

## Figma:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @yawsamcode    👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 



Highlights
✅All linters are checked
✅ Proper PR summary 
  
# Required Changes ♻️

- [x] I must admit you have really put in the effort so far in this project, however, the changes required has to do with a little bit of `CSS` adjustments


_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (yawsamcode) [reply index.html:224]: @Shedrack-Sunday Thanks for your feedback. I made effort to modify this to my best my only challenge was getting the background image to touch the border. I have set a padding round the page so I am not sure how to maneuver this at the moment. I believe if given the opportunity overtime I will figure it out. As for now time is running out 
challenge

image
1fdf2177184d.png)

File: index.html (hunk 2 above)
Comment (yawsamcode) [reply index.html]: After a long battle I was able to come close to the figma design. Its was though for me but I loved the challenge. 💪👊😆👊💪
see

File: index.html (hunk 3 above)
Comment (yawsamcode) [reply index.html:46]: I also tried to come up with the solution for this. After everything i feel like I just finished a boxing match. Let me go get a glass of juicer while I wait to hear from the all mighty code reviewer. Its a weekend so well all need to go get some fun. No more  coding for now.  Enjoy :) 😆😋😋😍
Capture

File: index.html (hunk 1 above)
Comment (mahbubul14) [reply index.html:224]: > * [x]  Good job working on this feature clap . However, the placement of the background image in your design needs a little adjustment to tally with that of the one given on the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=I0SUZ5FGkB27z6XP-0) Template. The reason for emphasizing this Is to help you build a programming muscle in meeting the requirements as given, which is the medium clients or your manager will follow. I've made a comparison below for your review
> 
> ## Figma
> image
> 
> ## Your design:
> image

- [ ] As the previous reviewer suggested, Kindly fix the issue. :+1:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @yawsamcode,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html [hunk 4]
```diff
@@ -45,119 +59,166 @@ <h2>I’m a software developer!</h2>
...
-
             </div>
-            <div class="see-project-btn" id="long-btn">See Project</div>
-            <div class="project-sample">
-                <div class="project-details">
-                    <h2>Profesional Art Printing Data</h2>
-                    <article>A daily selection of privately personalized reads; no accounts or sign-ups required.
-                        has been the industry's standard</article>
+
+            <div class="sub-project2">
```
Comment (amejid) [inline index.html:118]: - [ ] [OPTIONAL] I highly recommend you implement the changes suggested by the first reviewer. The image on the left indicates the state of the cards on hover. So every card should show the `See project` button when hovering over them. 😎️

image

**Review by amejid — APPROVED** [review]
Hi @yawsamcode, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 
- Good UI design :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**